Source/Core/Core/PatchEngine.cpp

```diff
@@ -196,9 +196,6 @@ static void ApplyPatches(const std::vector<Patch> &patches)
 				case PATCH_32BIT:
 					Memory::Write_U32(value, addr);
 					break;
-				default:
```

Comment:
I don't like how we currently silently fail and this change makes the assumption that we sanitize so it won't ever fail.
That would be the best to do so with whichever path you choose.